### ISSUE TITLE
Update Vet Centers to use va-icon

### ISSUE THREAD

**Issue by eselkin** [department-of-veterans-affairs/va.gov-cms#17874]
## User Story or Problem Statement

`<va-icon>` is replacing `<i>` font awesome tags. Vet Centers must replace font awesome with va-icon ASAP.

## Steps for Implementation

1. Change occurrences of `<i>` tags on Vet Center pages
    1. vets-website `src/applications/static-pages/facilities/FacilityMapSatelliteMainWidget.jsx` (1x)
    2. vets-website `src/applications/static-pages/facilities/FacilityMapWidget.jsx` (1x)
    3. vets-website `src/applications/static-pages/facilities/FacilityMapWidgetDynamic.jsx` (1x)
2. Change occurrences of `<i>` tags in Vet Center Locations pages
    1. vets-website `src/applications/static-pages/facilities/vet-center/components/ExpandableOperatingStatus.jsx` (2x)
       - Decision was made to update to use the proper component #17892 rather than only updating the icon
       - UPDATE 5/28: I have confirmed that all alerts (main, satellite and nearby) are using the proper component ([example](https://www.va.gov/st-paul-vet-center/locations/)) so but this file still has FA icons in it - is this no longer being used?

## Acceptance Criteria
- [x] Vet Center pages do not include fontawesome icons
- [x] Vet Center Locations Pages do not include fontawesome icons
- [x] Update and run existing test(s).
- [x] If additional tests are needed, stub a ticket to note this for future work.
- [x] Requires design review
- [x] Requires accessibility review

**Comment by eselkin** [comment]
@eselkin @laflannery Will add ticket for updating Nearby Vet Centers to use component for operating statuses. Will delete this file with the icon used.

**Comment by laflannery** [comment]
Question on ACs here - When we do component tickets we have ACs for tests:

> Update and run existing test(s).
If additional tests are needed, stub a ticket to note this for future work.

Should these 2 ACs be added to this ticket since we are saying that adding the Expandable alert component will be included in this ticket?

**Comment by eselkin** [comment]
adding ACs from @laflannery

**Comment by laflannery** [comment]
@eselkin does `src/applications/static-pages/facilities/FacilityMapWidget.jsx` also update the expand map icon on VAMC facility map icons? I just want to make sure we didn't miss that one

**Comment by randimays** [comment]
Successfully validated in prod on these three pages:
- https://www.va.gov/anoka-vet-center/: expand arrows icon on the mini map under `Locations and contact information`
- https://www.va.gov/anoka-vet-center/locations/: expand arrows icon on the mini map under `Locations` > `Main location`
- https://www.va.gov/brooklyn-vet-center/locations/: Regression check. We removed a component for an expandable alert that is no longer used and was replaced by one in content-build. This appears under `Harlem Vet Center`

AmandaK reviewed the icon we replaced in this Slack thread: https://dsva.slack.com/archives/C06NMUK12LR/p1717444422031389; Laura reviewed on the PR.

Closing!
